### ISSUE TITLE
Using large amounts of child entities for huge surface areas

### ISSUE THREAD

**Issue by elro0003** [tsunamayo/Starship-EVO#2755]
### Short Description of the issue:

each child entity has it's SA(surface area) calculated seperately. This can be exploited by filling an entire area with only pistons, hinges, rotors etc, giving the ship a very huge surface area, meaning the ship can be filled with engines weapons reactors and all that other stuff to make way too good ships.

### Steps to reproduce the issue:

Just stack a bunch of pistons in one area

### Environment
- Starship EVO version: 
- Graphics Card: 

### Additional information:

In my opinion we just need a way to calculate each seperate blocks/bricks SA instead of using a 1m grid to do it

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
Yes this is a know exploitable mechanic. What you are suggesting is very different though, and exploitable too, as it would mean that a 2x2 stretched hull would dissipate less heat than 2x2 separate hull blocks. Thanks
